lib/index.js

```diff
@@ -162,6 +164,31 @@ FSTree.prototype.calculatePatch = function(otherFSTree, isEqual) {
   return removals.reverse().concat(additions);
 };
 
+FSTree.applyPatch = function(source, output, patch) {
```

Comment:
I think this makes more sense if we're taking in an array of patches to apply. Otherwise, I'd argue it's simpler to just have a conditional outside the function. Thoughts? Probably makes sense for this to accept an array since `calculatePatch` returns an array anyway.